src/controller.cpp

```diff
@@ -124,6 +129,32 @@ int Controller::run(const CliArgsParser& args)
 		return args.return_code().value();
 	}
 
+	if (!can_open(configpaths.url_file())) {
+		if (!args.silent()) {
+			std::cout << strprintf::fmt(_("Starting %s %s..."),
```

Comment:
Does the program name and version really add much value to the user? Also we know at this point, that Newsboat will immediately exit for sure, so the "Starting" is a bit fishy in my point of view. In case you want to keep the program name and version how about "Cannot start %s %s: 'urls' file not found. Please…" and directly continue with the text below?

Reply:
I added this because the error message alone looked a bit weird to me. With this message, both the "happy path" and the error case begin with "Starting …", which is a nice uniformity.

But Newsbeuter migration doesn't print this either, and I agree with your arguments that it's fishy. So I guess I'll just remove this line altogether.

Reply:
> With this message, both the "happy path" and the error case begin with "Starting …", which is a nice uniformity.

Indeed, I never realized that.